sickbeard/scene_exceptions.py

```diff
@@ -273,9 +273,9 @@ def update_scene_exceptions(indexer_id, scene_exceptions, season=-1):
     # A change has been made to the scene exception list. Let's clear the cache, to make this visible
     if indexer_id in exceptionsCache:
         exceptionsCache[indexer_id] = {}
-        exceptionsCache[indexer_id][season] = scene_exceptions
+        exceptionsCache[indexer_id][season] = [se.decode('utf-8', 'ignore') for se in scene_exceptions]
```

Comment:
It should be unicode encoded right? Else you can't display the chinese chars right?
I personally think it's an issue with tornado, it is processing the post by default as encoded in str(). While it should be encoded in unicode.